### ISSUE TITLE
 Alpha loss is inf or nan. : Tensor had NaN values

### ISSUE THREAD

**Issue by mbed92** [mbed92/rl-physnet#3]
SAC agent returns NaN in a loss while executing:
```python 
loss_info = agent_learner.run(iterations=1)
```

It happens every time after the first iteration of training. Log:

```bash
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
[reverb/cc/client.cc:163] Sampler and server are owned by the same process (29021) so Table uniform_table is accessed directly without gRPC.
Traceback (most recent call last):
  File "/home/mbed/Projects/rl-physnet/scripts/train_haptic_rl.py", line 171, in <module>
    loss_info = agent_learner.run(iterations=1)
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tf_agents/train/learner.py", line 246, in run
    loss_info = self._train(iterations, iterator, parallel_iterations)
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tensorflow/python/eager/def_function.py", line 885, in __call__
    result = self._call(*args, **kwds)
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tensorflow/python/eager/def_function.py", line 956, in _call
    return self._concrete_stateful_fn._call_flat(
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tensorflow/python/eager/function.py", line 1963, in _call_flat
    return self._build_call_outputs(self._inference_function.call(
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tensorflow/python/eager/function.py", line 591, in call
    outputs = execute.execute(
  File "/home/mbed/anaconda3/envs/rl/lib/python3.8/site-packages/tensorflow/python/eager/execute.py", line 59, in quick_execute
    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,
tensorflow.python.framework.errors_impl.InvalidArgumentError:  Alpha loss is inf or nan. : Tensor had NaN values
	 [[node CheckNumerics_2 (defined at /anaconda3/envs/rl/lib/python3.8/site-packages/tf_agents/agents/sac/sac_agent.py:348) ]] [Op:__inference__train_1881479]

Errors may have originated from an input operation.
Input Source operations connected to node CheckNumerics_2:
 CheckNumerics_1 (defined at /anaconda3/envs/rl/lib/python3.8/site-packages/tf_agents/agents/sac/sac_agent.py:337)	
 mul_2 (defined at /anaconda3/envs/rl/lib/python3.8/site-packages/tf_agents/agents/sac/sac_agent.py:346)

Function call stack:
_train

[reverb/cc/platform/default/server.cc:84] Shutting down replay server
WARNING:absl:Writer-object deleted without calling .close explicitly.
[reverb/cc/writer.cc:382] Received error when closing the stream: [14] Socket closed

Process finished with exit code 1

```

**Comment by mbed92** [comment]
Might be related: tensorflow/agents#574

**Comment by mbed92** [comment]
Fixed by #4

**Comment by mbed92** [comment]
Still exists.
